projects/ngx-mapbox-gl/src/lib/map/map.service.ts

```diff
@@ -547,200 +633,145 @@ export class MapService {
   }
 
   private hookEvents(events: MapEvent) {
-    this.mapInstance.on('load', () => {
+    this.mapInstance.on('load', (evt) => {
       this.mapLoaded.next(undefined);
       this.mapLoaded.complete();
-      this.zone.run(() => events.load.emit(this.mapInstance));
+      this.zone.run(() => events.load.emit(evt.target));
```

Comment:
evt.target === this.mapInstance on mapbox-gl 1.1.0
Ok

Reply:
Right, I initially changed it since I wanted to emit `evt` but that was going to be a breaking change. Using `evt.target` to be safe, but it should always equal `this.mapInstance`.